cross/libtorrent-rakshasa/Makefile

```diff
@@ -14,7 +14,7 @@ LICENSE  = GPLv2
 
 GNU_CONFIGURE = 1
 PRE_PATCH_TARGET = myPrePatch
-CONFIGURE_ARGS = --disable-instrumentation
+CONFIGURE_ARGS = --disable-instrumentation --with-zlib=$(STAGING_INSTALL_PREFIX)
```

Comment:
You should add `cross/zlib` to depends as well. Even though it comes along with openssl, better to be explicit.